dev-bin/make-man-pages.pl

```diff
@@ -41,7 +40,7 @@ sub _make_man {
 
     my $input   = "$Bin/../doc/$name.md";
     my $man_dir = "$target/man/man$section";
-    mkpath($man_dir);
+    mkpath($man_dir) or die "Failed to create directory $man_dir: $!";
```

Comment:
Possibly `mkpath` handles errors already as well, although its docs seem more ambiguous to me!